modules/modes/drag_node.js

```diff
@@ -232,6 +232,9 @@ export function modeDragNode(context) {
             isInvalid = hasInvalidGeometry(entity, context.graph());
         }
 
+        if (entity.type === 'node' && entity.isLocked()) {
```

Comment:
Would it be feasible to allow the drag when some modifier key is held down, like Alt/Option, when the mapper is sure they know what they’re doing? If not, another approach would be to create a filter for things like survey points and hide it by default, as we do with boundaries.

/ref https://github.com/openstreetmap/iD/issues/9255#issuecomment-1224954446